desk/app/channels-server.hoon

```diff
@@ -360,16 +360,16 @@
       =/  seq
         %+  skim
           seq.action
-        |=  =id:h
-        (~(has by hooks.hooks) id)
+        ~(has by hooks.hooks)
       =.  order.hooks  (~(put by order.hooks) nest.action seq)
-      (give-hook-response %order nest.action seq)
+      =/  =response:h  [%order nest.action seq]
+      (give %fact ~[/v0/hooks] hook-response-0+!>(response))
```

Comment:
This risks diverging from `+ho-give-response`. It might've been fine to keep a `+give-hook-response` at the top level here if `+ho-give-response` was written to invoke that instead of doing a `+give` (which turns into a top-level `+give` modification of `cor` anyway).

Not a huge deal, I'm not opposed to hand-written cards, but this code may get more complex in the presence of additional protocol versions.

Reply:
yeah I will leave for now, but yeah we should probably have an overarching `give-hook-response`